### DISCUSSION TITLE
Schmid Factor results for quartz

### DISCUSSION THREAD

**Discussion by DJMUNLV** [mtex-toolbox/mtex#1715] [Ask Anything] [answered]
**Hello, 
I am looking to calculate the Schmid factor for quartz slip systems. Here is the code I've used and the results are below. I am confused as MTEX outputs SF twice.  I expected the 0.0306   -0.1367    0.0306 for each symmetrically equal plane in the system but what is the first really small value? And are there supposed to be two SF outputs as seen below? Thank you.**

CS = crystalSymmetry('321', [4.88,4.88,5.36], 'X||a*', 'Y||b', 'Z||c*', 'mineral', 'Quartz', 'color', 'light blue');

% define a slip system 
b=Miller(-1,-1,2,0, CS)
n=Miller(0,0,0,1,CS)
sS = slipSystem(b, n)

% define a loading direction
loadingDirection = Miller(0,0,0,1,CS)

% compute the Schmid factor for the slip system sS
SF = SchmidFactor(sS,loadingDirection)

% compute the Schmid factor for all slip systems symmetrically equivalent  to sS
SF = SchmidFactor(sS.symmetrise,loadingDirection)

**ANSWER** 

SF =

   3.0616e-17

SF =

   1.0e-15 *

    0.0306   -0.1367    0.0306

**Comment by kilir** [comment, marked as answer]
Hi,
not sure if I understand your question, but first you simply compute the Schmid factor for `sS`, and then for the three symmetrically equivalent slip systems. And 3.0616e-17 is pretty much the same as 0.0306e-15.

And the reason they are all pretty much 0 is that you are loading normal to your slip plane.

Hope that helps.
Cheers,
Rüdiger

**Comment by DJMUNLV** [comment]
Hello Rudiger, 

I see, so SF = 3.0616e-17 is the Schmid for the sS given while 

SF = 1.0e-15 *
0.0306   -0.1367    0.0306

are the values of those symmetrically equivalent. Thanks for your help!
